github/src/main/kotlin/gropius/sync/github/IssueGrabber.kt

```diff
@@ -0,0 +1,117 @@
+package gropius.sync.github
+
+import com.apollographql.apollo3.ApolloClient
+import gropius.sync.github.generated.IssueReadQuery
+import gropius.sync.github.generated.IssueReadQuery.Data.Companion.metaData
+import gropius.sync.github.generated.IssueReadQuery.Data.Repository.Issues.PageInfo.Companion.pageInfoData
+import gropius.sync.github.generated.fragment.IssueDataExtensive
+import gropius.sync.github.model.IssueDataCache
+import gropius.sync.github.model.RepositoryInfo
+import gropius.sync.github.repository.RepositoryInfoRepository
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.map
+import kotlinx.coroutines.reactive.asFlow
+import kotlinx.coroutines.reactor.awaitSingle
+import org.bson.types.ObjectId
+import org.springframework.data.mongodb.core.*
+import org.springframework.data.mongodb.core.query.Criteria
+import org.springframework.data.mongodb.core.query.Criteria.where
+import org.springframework.data.mongodb.core.query.Query
+import org.springframework.data.mongodb.core.query.Query.query
+import org.springframework.data.mongodb.core.query.Update
+import org.springframework.data.mongodb.core.query.Update.update
+import java.time.OffsetDateTime
+
+/**
+ * Implementation of Grabber to retrieve issues and cache them in the database
+ */
+class IssueGrabber(
+    private val repositoryInfoRepository: RepositoryInfoRepository, private val mongoOperations: ReactiveMongoOperations
+) : Grabber<IssueDataExtensive>() {
+
+    /**
+     * test repository user
+     */
+    val user = "espressif"
+
+    /**
+     * test repository name
+     */
+    val repo = "idf-component-manager" //"llvm-project"
```

Comment:
what about this inline comment?

Reply:
an alternative test repo. I use this to switch between many issues and few issues until you finished your mutations